docs/onboarding/Database-Structure.md

```diff
@@ -71,19 +71,20 @@ Stores onboarding OTP codes used during activation and user verification.
 
 | Name | Type | Info | Note |
 |---|---|---|---|
-| `id` | `VARCHAR(36)` | `NOT NULL PRIMARY KEY` | Autogenerated record identifier (UUID). |
-| `process_id` | `VARCHAR(36)` | `NOT NULL` | Process identifier (UUID). |
-| `otp_code` | `VARCHAR(32)` | `NOT NULL` | Generated OTP code. |
-| `status` | `VARCHAR(36)` |  | Generated OTP code. |
-| `type` | `VARCHAR(32)` | `NOT NULL` | OTP code type (`ACTIVATION`, `USER_VERIFICATION`). |
-| `error_detail` | `VARCHAR(256)` |  | Detail of error (e.g. information about timeout or exceeded number of failed attempts). |
-| `error_origin` | `VARCHAR(256)` |  | Origin of the error (`DOCUMENT_VERIFICATION`, `PRESENCE_CHECK`, `CLIENT_EVALUATION`, `OTP_VERIFICATION`, `PROCESS_LIMIT_CHECK`, `USER_REQUEST`). |
-| `failed_attempts` | `INTEGER` |  | Number of failed attempts for verification, |
-| `timestamp_created` | `TIMESTAMP` | `NOT NULL DEFAULT CURRENT_TIMESTAMP` | Timestamp when process was started. |
-| `timestamp_expiration` | `TIMESTAMP` | `NOT NULL` | Timestamp when the OTP expires. |
-| `timestamp_last_updated` | `TIMESTAMP` |  | Timestamp when record was last updated. |
-| `timestamp_verified` | `TIMESTAMP` |  | Timestamp when OTP was verified. |
-| `timestamp_failed` | `TIMESTAMP` |  | Timestamp when OTP failed. |
+| `id`                     | `VARCHAR(36)` | `NOT NULL PRIMARY KEY` | Autogenerated record identifier (UUID).                                                                                                          |
```

Comment:
My idea was to reformat all columns of this single table or do not reformat it at all.